modules/health_quest/app/services/health_quest/pgd/fhir_client.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module Pgd
```

Comment:
Could this be spelled out as `PatientGeneratedData`?

Reply:
I thought of doing that in the beginning but it's known as PGD everywhere in the domain and by everyone and acronyms seemed to be the norm for domain level objects/concepts. Is there a push to move away from this convention?

Reply:
We're trying to think more about acronyms.  As we move to using the Rails 6 zeitwerk loader, acronyms have impacts on how files are loaded.  As an acronym it should be `PGD`, but if we're taking that approach there should be an entry in `config/initializers/inflections.rb`.  But since that's a universal acronym file, I'm thinking we should be more conservative on what becomes an acronym.  While PGD is known in the domain, it's less obvious from the larger `vets-api` perspective.  We're trying to get zeitwerk across the line before we formalize an approach, so feedback is welcome.  My comment came from a place where we've been running into a lot of these acronyms we're not sure which way to break on, and PGD is not in the `/wtf` slackbot, so I assumed it's not commonly used across the VA.

Reply:
I think I'll go with PatientGeneratedData as you suggested. I agree that we should have a larger conversation about acronym usage in the codebase in general, and I think we should potentially involve a lot more people in that conversation. Personally, I think I'd rather avoid using acronyms in any codebase; Just like I'd avoid using a single character variable.

Reply:
Done and ready for re-review!